pom.xml

```diff
@@ -86,11 +86,6 @@
             <name>osiam-releases</name>
             <url>https://api.bintray.com/maven/osiam/OSIAM/org.osiam:osiam/;publish=1</url>
         </repository>
-        <snapshotRepository>
-            <id>snapshots</id>
-            <name>osiam-snapshots</name>
-            <url>https://oss.jfrog.org/artifactory/oss-snapshot-local</url>
-        </snapshotRepository>
```

Comment:
Also, the build fails:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy (default-deploy) on project osiam: Failed to deploy artifacts: Could not transfer artifact org.osiam:osiam:war:3.0-20160608.081458-1 from/to releases (https://api.bintray.com/maven/osiam/OSIAM/org.osiam:osiam/;publish=1): Failed to transfer file: https://api.bintray.com/maven/osiam/OSIAM/org.osiam:osiam/;publish=1/org/osiam/osiam/3.0-SNAPSHOT/osiam-3.0-20160608.081458-1.war. Return code is: 400, ReasonPhrase: Bad Request. -> [Help 1]
```

Maybe we have to disable the deploy plugin for snapshot builds?